### DISCUSSION TITLE
Close expanded row when expanding other row.

### DISCUSSION THREAD

**Discussion by kumaranubhaw** [carbon-design-system/carbon#12059] [Help] [answered]
I have list of rows in DataTable component. On clicking row I am fetching data for expanded view.
I want to close other expanded row when I click to expand current row.
Please help me, how can I achieve that.

**Comment by jnm2377** [comment, marked as answer]
Hey @kumaranubhaw I think it is certainly possible to do this, though it will require a bit of custom logic in your implementation. I think you'll likely want to use the `isExpanded`  and `onExpand` props, where `onExpand` you have some sort of logic that determines what other rows have the `isExpanded` prop and set that to false. 

<img width="1068" alt="Captura de Pantalla 2022-09-07 a la(s) 9 32 48 p m" src="https://user-images.githubusercontent.com/32556167/189020572-0fb33484-df15-49d8-aaee-8117e157b493.png">

**Reply by kumaranubhaw** [reply]
@jnm2377 Thank you for the help. It's working as expected on using `onExpand ` and `isExpanded `!!
